docs/index.md

```diff
@@ -51,6 +51,16 @@ val app = a <> b
 
 Apps can be composed using the `<>` operator. The way it works is, if none of the routes match in `a` , or a `NotFound` error is thrown from `a`, and then the control is passed on to the `b` app.
 
+# ZIO integration
+
+```
+val app: Http[Any, HttpError, Request, Response] = Http.collectM[Request] {
+  case Method.GET -> Root / "hello" => ZIO.succeed("Hello World").map(msg => Response.text(msg))
```

Comment:
```ZIO.succeed("Hello world").map( msg => Response.text(msg))``` can be replaced with ```ZIO.succeed(Response.text("Hello world"))```

Reply:
The goal was to be more like a "real life" example, but it would be simpler like this indeed

Reply:
done :)